src/users/users.controller.ts

```diff
@@ -33,3 +39,23 @@ export class UsersController {
   }
 }
 
+@Controller('user') 
+export class CurrentUserController {
+  constructor(private readonly usersService: UsersService) {}
+
+  @Get('') 

```

Comment:
```suggestion
  @Get() 
```

với empty thì viết như trên là đủ r em nhé,
ko cần chua thêm string rỗng